blockchain_connector/docker/Dockerfile-fabric.centos

```diff
@@ -0,0 +1,186 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+FROM centos:centos8 as base_image
+
+# Add necessary packages
+RUN yum update -y \
+ && yum install -y \
+    ca-certificates \
+    python3-requests \
+    python3-pip \
+ && yum clean all
+
+RUN pip3 install toml colorlog twisted
+
+# Make Python3 default
+RUN ln -sf /usr/bin/python3 /usr/bin/python
+
+# -------------=== python build ===-------------
+
+#Build python intermediate docker image
+FROM centos:centos8 as python_image
+
+# Add necessary packages
+RUN yum update -y\
+ && yum install -y \
+    ca-certificates \
+    pkg-config \
+    python3-pip \
+    python3-devel \
+    make \
+ && yum clean all
+
+# Install setuptools packages using pip because
+# these are not available in apt repository.
```

Comment:
Replace "apt" with "the". Apt is for Debian/Ubuntu, not CentOS.

Reply:
Done